.github/workflows/deploy-zeit-staging.yml

```diff
@@ -66,23 +66,40 @@ jobs:
           ZEIT_TOKEN: ${{ secrets.ZEIT_TOKEN }} # Passing github's secret to the worker
           CURRENT_BRANCH: ${{ github.ref }} # Passing current branch to worker
 
+      # We need to find the PR id to be able to comment it.
+      # It avoids, on many push at the same time, issues about the pull request (HEAD moving).
+      - name: Finding Pull Request ID
+        uses: jwalton/gh-find-current-pr@v1
+        id: pr_id_finder
+        if: always() # We want to get the PR ID, even when it fails.
```

Comment:
Even if "what" fails? Does that action run even if the previous fails? I'm not sure to understand well.

Reply:
If the deployment or any command doesn't work, the job is "in failure" and doesn't run the other jobs.
But in case of failure, we want the ID to put in in the URL.
Does it need more documentation in the workflow ?

Reply:
`We always need to get the PR ID, so that we may comment on the PR, wether the previous action failed or succeed.`